renovate.json

```diff
@@ -35,11 +32,17 @@
         "platformAutomerge": true,
         "schedule": [
             "at any time"
+        ],
+        "packageRules": [
+            {
+                "automerge": true,
+                "matchUpdateTypes": [
+                    "digest"
+                ]
+            }
```

Comment:
Missing the `addLabels` field to allow auto merging of tekton digest 
prs
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/renovate.json#L59

Reply:
I though automerge is enough based on: https://konflux.pages.redhat.com/docs/users/mintmaker/user.html#automerge:~:text=%22packageRules%22%3A%20%5B%0A%20%20%20%20%7B%0A%20%20%20%20%20%20%22description%22%3A%20%22Automerge%20non%2Dmajor%20updates%22%2C%0A%20%20%20%20%20%20%22matchUpdateTypes%22%3A%20%5B%22minor%22%2C%20%22patch%22%5D%2C%0A%20%20%20%20%20%20%22automerge%22%3A%20true%0A%20%20%20%20%7D
it makes sense to add the labels though, thanks for highlighting this.

Reply:
The documentation is still pending some changes in https://issues.redhat.com/browse/KONFLUX-8240